index.js

```diff
@@ -1944,11 +1979,16 @@
     });
   }
 
-  //  invalidArgumentsLength :: (String, Integer, Integer) -> Error
-  function invalidArgumentsLength(name, expectedLength, actualLength) {
-    return new TypeError(q(name) +
-                         ' requires ' + numArgs(expectedLength) + ';' +
-                         ' received ' + numArgs(actualLength));
+  //  tooManyArguments :: (TypeInfo, Integer) -> Error
```

Comment:
can we use `invalidArgumentsLength` instead of adding this `tooManyArguments` function?